toolchain/check/import_ref.cpp

```diff
@@ -1180,6 +1206,8 @@ class ImportRefResolver {
         context_.AddPlaceholderInstInNoBlock(SemIR::LocIdAndInst(
             AddImportIRInst(import_interface.decl_id), interface_decl));
 
+    // TODO: Support for importing generics.
```

Comment:
Where you have these TODOs, would it be worth a `context.TODO` to produce a diagnostic? Or do you basically already have this implemented?

Reply:
I don't have a full implementation yet, but this comes up in the prelude so adding a `context.TODO` would break every test. I'll be fixing this in the short term.